client/src/theme/Components/reader/_appearance-menu.scss

```diff
@@ -126,4 +127,64 @@
       margin-left: 20px;
     }
   }
+
+  .control-margins {
+    button {
+      @include buttonUnstyled();
+      width: 120px;
+      height: 60px;
+      background-color: $neutralWhite;
+      transition: all $duration $timing;
+
+      &.margin-increase {
+        .compound-icon {
+          display: inline-block;
+          width: 83px;
+          height: 23px;
+          position: relative;
+          i {
+            position: absolute;
+            font-size: 28px;
+            left: 0;
+            top: 0;
+          }
+        }
+      }
+
+      &.margin-decrease {
+        .compound-icon {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4